config/permissions.tmpl.yml

```diff
@@ -36,6 +36,7 @@
 - ems-type:openstack_infra
 - ems-type:openstack_network
 - ems-type:physical_infra_manager
+- ems-type:lenovo_ph_infra
```

Comment:
I'm kidding of course...I see you were trying to group it with the phys_infra_manager, but a) the file is alphabetized regardless, and b) that line item should no longer exist.  I believe it was added temporarily until the first subclass was created, which is now.  I would expect that this PR would remove the physical_infra_manager.  git-blame shows it was added in https://github.com/ManageIQ/manageiq/commit/7fb40688b4401f772490ec18f1e011660d51cd0b